### DISCUSSION TITLE
Time Tracking suggestion: Brewing

### DISCUSSION THREAD

**Discussion by ArashiJoe** [runelite/runelite#15991] [Ideas]
Add a timer for brewing ale/cider in Keldagrim and another timer for brewing in Port Phasmatys.
Brewing follows the hardwood farming cycle, so it should be easy to implement. The only major variable is whether or not "The Stuff" is used to increase the chance for ales to mature. (Would be super cool if there was an indication as to if your ale matured or not, similar to how crops will appear as dead if they do indeed die. Not sure if this is possible though.)

**Comment by Nightfirecat** [comment]
The [Brewing hub plugin](https://runelite.net/plugin-hub/show/brewing) basically does this already.

**Reply by rigadoog** [reply]
> The [Brewing hub plugin](https://runelite.net/plugin-hub/show/brewing) basically does this already.

It's not as useful as having brewing integrated into the Time Tracking plugin :/ as the OP said it runs on the same cycle as hardwood/anima patches
